x-pack/test/apm_api_integration/tests/throughput/service_maps.spec.ts

```diff
@@ -113,7 +121,24 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             [
               ...Object.values(throughputTransactionValues),
               ...Object.values(throughputMetricValues),
-            ].forEach((value) => expect(roundNumber(value)).to.be.equal(roundNumber(GO_DEV_RATE)));
+            ].forEach((value) =>
+              expect(roundNumber(value)).to.be.equal(roundNumber(GO_DEV_RATE + GO_PROD_RATE))
```

Comment:
I don't know, that was making the test fail after my changes. But was passing before, we took a look yesterday with @cauemarcondes and fix it.

Reply:
ah, because transaction type was `Worker` before. I think that was intentional, to make sure that this only included transaction stats for transaction type request or page-load. Is there a reason why you removed that here?

Reply:
Yeah, that right, it should filter out the transaction if transaction type is 'Worker', I'm going to take a look a see what's going on there.